flintrock/util.py

```diff
@@ -0,0 +1,18 @@
+import os
+import sys
+
+FROZEN = getattr(sys, 'frozen', False)
+
+
+def get_subprocess_env() -> dict:
+    """
+    Get the environment we want to use when making subprocess calls.
+    This takes care of details that affect subprocess calls made from
+    PyInstaller-packaged versions of Flintrock.
+
+    For more information see: https://github.com/pyinstaller/pyinstaller/blob/develop/doc/runtime-information.rst#ld_library_path--libpath-considerations
+    """
+    env = dict(os.environ)
+    if FROZEN and env.get('LD_LIBRARY_PATH_ORIG') is not None:
+        env['LD_LIBRARY_PATH'] = env['LD_LIBRARY_PATH_ORIG']
```

Comment:
According to [this comment](https://github.com/pyinstaller/pyinstaller/issues/2237#issuecomment-258471132) I may need to change this so that `LD_LIBRARY_PATH` gets set to an empty string if `LD_LIBRARY_PATH_ORIG` is not defined.